src/src.builds

```diff
@@ -2,9 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectExclusions Condition="'$(OS)' != 'Windows_NT'" Include="$(MSBuildThisFileDirectory)**\src\System.Memory.csproj" />
```

Comment:
I simply wanted to align it with the other projects.  Seems reasonable to have this in configurations, though perhaps a comment should be added.  I don't have a strong preference, I just cleaned it up because of a side comment Wes made in a different PR.